src/main/java/com/github/tomakehurst/wiremock/client/WireMock.java

```diff
@@ -464,6 +503,41 @@ public static UrlPathPattern urlPathTemplate(String pathTemplate) {
     return new UrlPathTemplatePattern(pathTemplate);
   }
 
+  public static MultiValuePattern havingExactly(final StringValuePattern... valuePatterns) {
+    if (valuePatterns.length == 0) {
+      return noValues();
+    }
+    return new ExactMatchMultiValuePattern(Stream.of(valuePatterns).collect(Collectors.toList()));
+  }
+
+  public static MultiValuePattern havingExactly(String... values) {
+    if (values.length == 0) {
+      return noValues();
+    }
+    return new ExactMatchMultiValuePattern(
+        Stream.of(values).map(EqualToPattern::new).collect(Collectors.toList()));
+  }
+
+  public static MultiValuePattern including(final StringValuePattern... valuePatterns) {
+    if (valuePatterns.length == 0) {
+      return noValues();
+    }
+    return new IncludesMatchMultiValuePattern(
+        Stream.of(valuePatterns).collect(Collectors.toList()));
+  }
+
+  public static MultiValuePattern including(String... values) {
```

Comment:
This method could delegate to the overload above it rather than replicating its logic.

Reply:
yea... makes sense. I forgot about the same.